crates/rolldown/src/module_finalizers/scope_hoisting/impl_visit_mut.rs

```diff
@@ -112,25 +110,10 @@ impl<'me, 'ast> VisitMut<'ast> for ScopeHoistingFinalizer<'me, 'ast> {
               Module::External(importee) => {
                 match self.ctx.options.format {
                   rolldown_common::OutputFormat::Esm => {
-                    // Insert `import * as ns from 'ext'`
-                    // Insert `__reExport(exports, ns)`
-                    let re_export_fn_name = self.canonical_name_for_runtime("__reExport");
-                    let importer_namespace_name =
-                      self.canonical_name_for(self.ctx.module.namespace_object_ref);
                     let importee_namespace_name = self.canonical_name_for(rec.namespace_ref);
```

Comment:
You could get this in `EcmaModule#star_exports` .

Reply:
I tweak a little, but I still can't find a way to get `rec.namespace_ref` without get `rec`